docs/index.rst

```diff
@@ -7,11 +7,12 @@ Welcome to astro-sdk's documentation!
 =====================================
 
 .. toctree::
+   CHANGELOG.md
+   CONTRIBUTING.md
    :maxdepth: 2
    :caption: Contents:
 
 
-
 Indices and tables
 ==================
 
```

Comment:
```suggestion
Indices and tables
==================

    * :ref:`genindex`
    * :ref:`modindex`
    * :ref:`search`
```
This would enable quick search on indexes as well.